### ISSUE TITLE
Error: master when trying to use the CLI

### ISSUE THREAD

**Issue by guenter** [mesos/kafka#11]
I'm getting this when trying any of the command line action:

```
tobi@Zwerg ~/code/kafka (master)$ ./kafka-mesos.sh add 0
Error: master
```

A better errors message would be great!

**Comment by joestein** [comment]
check your kafka-mesos.properties and make sure you have "master" routable as a host or change the property to match zk or master to connect too

**Comment by LLParse** [comment]
Opened https://github.com/mesos/kafka/pull/13.

**Comment by dmitrypekar** [comment]
Merged #13
